instrumentation/jms/src/main/java/brave/jms/PropertyFilter.java

```diff
@@ -74,6 +80,32 @@ static void filterProperties(Message message, Set<String> namesToClear, List<Obj
       return;
     }
 
+    // workaround for ActiveMQBytesMessage
+    if (message instanceof BytesMessage) {
```

Comment:
```suggestion
    if (message instanceof BytesMessage) { // BytesMessage requires clearing the body before reset properties otherwise it fails with MessageNotWriteableException
```